src/pages/tabs/tabs.ts

```diff
@@ -104,12 +104,13 @@ export class TabsPage {
         'transactionProposalRemoved',
         'TxProposalRemoved',
         'NewOutgoingTx',
-        'UpdateTx'
+        'UpdateTx',
+        'NewIncomingTx'
       ],
       (eventName: string) => {
         if (walletId && type == eventName) {
           setTimeout(() => {
-            this.updateTxps();
+            this.fetchAllWalletsStatus();
```

Comment:
I wouldn't like to fetch all wallets when happen any of those events (performance issue). If it's only for new incoming tx I'd rather to a new function